### PR TITLE
Controllers no longer had to add kcp.dev/cluster labels

### PR DESCRIPTION
**Pull request by guillaumerose**
By using the new namespace scheduler.

When a ns is created, it is assigned to a cluster. Objects are automatically synced.

### CODE REVIEW

**Review by bnallapeta — APPROVED** [review]
Works locally with these changes. Although, the issue with continuous syncing of 'Resource=leases' exist.

File: run.sh [hunk 1]
```diff
@@ -135,12 +135,10 @@ kubectl get pods,taskruns,pipelineruns
 # Add a secret in the physical cluster so that the event listener and interceptors can query KCP API
 cp ./.kcp/admin.kubeconfig ./.kcp/remote.kubeconfig
 gsed -i "s/\[::1\]/host.docker.internal/" ./.kcp/remote.kubeconfig
```
Comment (bnallapeta) [inline run.sh:137]: gsed does not work on fedora. Had to change it to sed.

Comment (vdemeester) [reply run.sh:137]: Ahhh. OSX users 😝

Comment (guillaumerose) [reply run.sh:137]: yes sorry. If someone knows how to make it transparent, I am happy to configure my machine. :D

Comment (vdemeester) [reply run.sh:137]: @chmouel ^^

Comment (chmouel) [reply run.sh:137]: i think if you prepend the path in your .zshrc with the gnubin paths from brew and it should works: 

at least it does for me, ie i have this :

```
path=(/opt/homebrew/opt/*/libexec/gnubin
      /opt/homebrew/opt/file-formula/bin
      /opt/homebrew/sbin
      /opt/homebrew/bin $path) #prepend
```

Comment (guillaumerose) [reply run.sh]: Fixed, thanks :)

**Pull request merged**